Sources/SingleFactorAuth/Web3AuthOptions.swift

```diff
@@ -1,20 +1,35 @@
 import FetchNodeDetails
 
-public class Web3AuthOptions {
+public class Web3AuthOptions: Codable {
     private var web3AuthNetwork: Web3AuthNetwork
     private var clientId: String
     private var sessionTime: Int
     private var storageServerUrl: String? = nil
     private var serverTimeOffset: Int = 0
+    var whiteLabel: WhiteLabelData? = nil
+    var originData: [String: String]?
+    var buildEnv: BuildEnv = .production
+    var redirectUrl: String?
+    var walletSdkUrl: String?
+    var chainConfig: ChainConfig? = nil
```

Comment:
Any reason for having chainConfig in options?

Reply:
Same sturucture as of PnP.